### ISSUE TITLE
iCalcreator incompatible with PHP v5.6?

### ISSUE THREAD

**Issue by dleffler** [iCalcreator/iCalcreator#72]
When I attempt to load (require_once) iCalCreator 2.26.8 or 2.27.x under PHP V5.6.x, I get the following fatal error (release notes still state v5.6 compatible)
```
 Parse error: syntax error, unexpected '->' (T_OBJECT_OPERATOR) in iCalcreator-2.26.8\src\Util\Util.php on line 1994
```
Not sure if this issue was prevalent in previous versions 2.24.0 or 2.22.5

**Comment by iCalcreator** [comment]
It is bug in 2.26.8, no line 1994 in 2.27.x.

Working on a fix right now!

**Comment by iCalcreator** [comment]
Typo error, fixed in 2.26.9
